test/controlplane/suite/testcase.go

```diff
@@ -164,7 +163,7 @@ func (cpt *ControlPlaneTest) StartOperator(modCellConfig func(vp *viper.Viper))
 			cell.Provide(func() k8sClient.Clientset {
 				return cpt.clients
 			}),
-			operatorCmd.OperatorCell,
+			// operatorCmd.OperatorCell,
```

Comment:
Remove?

Reply:
This can actually be changed to `operatorCmd.ControlPlane` in the first commit, I'm gonna fix :+1: